automation/README.md

```diff
@@ -58,20 +58,20 @@ Now, simply open the test spec, right-click on the class name or a specific test
 To run all tests:
 
 ```bash
-sbt test -Djsse.enableSNIExtension=false -Dheadless=false
+sbt test -Dheadless=false
 ```
 
 To run a single suite:
 
 ```bash
-sbt -Djsse.enableSNIExtension=false -Dheadless=false "testOnly *GoogleSpec"
+sbt -Dheadless=false "testOnly *GoogleSpec"
 ```
 
 To run a single test within a suite:
 
 ```bash
 # matches test via substring
-sbt -Djsse.enableSNIExtension=false -Dheadless=false "testOnly *GoogleSpec -- -z \"have a search field\""
+sbt -Dheadless=false "testOnly *GoogleSpec -- -z \"have a search field\""
```

Comment:
I think `headless` is only used by Selenium too, and we could remove that flag as well.

Reply:
Removed.